src/main/java/org/broadinstitute/dsm/cf/TestBostonKitTrackerDispatcher.java

```diff
@@ -84,7 +85,10 @@ public void accept(PubsubMessage pubsubMessage, Context context) throws Exceptio
                     int lastKitId = 0;
                     UPSKit kit = null;
                     logger.info("tracking ups ids for " + ddpInstance.getName());
-                    boolean gbfShippedTriggerDSSDelivered = InstanceSettings.getInstanceSettings(ddpInstance.getName(), conn).isGbfShippedTriggerDSSDelivered();
+                    InstanceSettingsDto instanceSettings = new InstanceSettings().getInstanceSettings(ddpInstance.getName());
```

Comment:
I think that won't work for CF. you need to pass the same connection so it knows how to connect to DB and get the instance settings

Reply:
@pegahtah does it use different database or?